request-manager/graph/grapher.go

```diff
@@ -393,5 +444,9 @@ func getMissingArgs(n *spec.Node, jobArgs map[string]bool) ([]string, error) {
 		}
 	}
 
-	return missing, nil
+	if len(missing) > 0 {
+		return fmt.Errorf("node %s missing job args: %s", n.Name, strings.Join(missing, ", "))
```

Comment:
These jobs aren't "missing", they're not set. This slight change in wording makes a big difference because node specs have a `sets` field. So errors says "job args aren't set" clues the user into the problem: these args are in some node's `sets` list.